velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -557,6 +582,75 @@ TEST_F(ArithmeticTest, widthBucket) {
   EXPECT_EQ(widthBucket(-kInf, 0, 4, 3), 0);
 }
 
+TEST_F(ArithmeticTest, checkAdd) {
+  std::string func = "check_add";
```

Comment:
nit: maybe add a lambda function `assertErrorForCheckedAdd` to avoid using the function name repeatedly.

Reply:
Looks we can't use template parameter with lambda function in c++17, I extract them as functions.
Thanks.